data/changelog.txt

```diff
@@ -696,848 +605,226 @@ Cancel out of crafting menu during component selection.
-Explanation for the player when choosing the direction for the new headlight
-forbids filling of tanks with rotten liquids
-Show "Time: ???" when underground and without watches
-Moves ethanol to after disinfectant in recipes
-Add FIRE flag to lit oil lamp
-Fix message window overlapping border in advanced inventory UI
-Fix revealing invisible monster when looking around
-Fix season names not being detected by xgettext
-Add error sound effect to main menu and use menu change sound effect for options
-Some assorted scrollbar related fixes
-Don't cancel activity if electricity-immune player is struck by it
-Road-Runner and Slow Footed cancel each other
-Enable to customize 16 ANSI color slots
-Hide bionic slots for 0.D
-Fix reload failure message
-Chesthole, 32, Fixes boarded up windows, broken down doors, adds the bitten effect
-Info about conductivity now mentions item, not weapon
-Sleep-through messages
-Message Window Makeover
-Restore max range to targeting and item info.
-Fix issues in #21173
-Adds consistent usage of caliber to rockets
-Added more descriptive text to ammo belt names
-update reloaded_50_incendiary description to match 50_incendiary description
-Removed references to weight torque and recoil from M134D-H
-Changed capitalization of disarm technique
-Added disclaimer to tutorial section
-Record NPC kills in kill count
-Handle music volume change
-Makes holsters display their minimum volume
-Sort preset templates
-Clearer dialog on Tracker Mission
-Some small spelling and spacing fixes
-NPC-friendly messages for dropping items
+Enable to customize 16 ANSI color slots.
 Add option to toggle framebuffer acceleration when using software rendering.
-Small text update for washboard use
-Fix signs message in extended description
-Talk to NPCs from the menu for examining them
+Talk to NPCs from the menu for examining them.
 Added ability to save and restore default layout for advanced inventory.
-[minor]Fix two broken wiki link
-Renamed Vacutainer© to Blood draw kit.
-Forgot to recalc panes when restoring advanced inventory layout.
-In the crafting GUI, show which books provide this recipe
-Added point pool restriction option in world generation settings
-Customize Consume Item menu
-Added more informative text when trying to install bionics depending on player's first aid skill
-No exact numbers of damage on NPC attack
+In the crafting GUI, show which books provide this recipe.
+Added point pool restriction option in world generation settings.
 Added monster info in extended description.
-Fix player lifetime stats
-Smarter animation delays + UI popups with a "hourglass"
-Add default book recipe difficulty instead of requiring it to be specified every time
-Forbid options for waiting till X when underground
-Overmap overhaul (step 9)
-Some enhancements and fixes to pixel minimap
-Add deconstruction recipes for missing ammo types
-Display actual nutrition acquired, rework rotten food penalties
-Ranch border visible on overmap
```

Comment:
This shouldn't be deleted.

Reply:
The ranch was added in this version, so individual entries regarding it it are IMO unecessary.

Reply:
Fair enough.  Request withdrawn.